lib/reek/ast/sexp_extensions.rb

```diff
@@ -442,6 +442,19 @@ def method_name
       end
 
       ZsuperNode = SuperNode
+
+      # Utility methods for :sym nodes.
+      module SymNode
+        def name
+          children.first
+        end
+
+        # FIXME: Move full_name logic to contexts
```

Comment:
Let's remove this comment and rather discuss it via github issue. Comments like that tend to get stale and confusing pretty quick....